### PR TITLE
Julian Example Endpoint

### PR DESCRIPTION
**Pull request by JulianKorgol**

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
 from starlette.applications import Starlette
 from starlette.responses import JSONResponse
 from starlette.routing import Route
+from datetime import datetime
```
Comment (t1waz) [inline backend/main.py]: imports should be sorted

File: backend/main.py [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 
 
 async def homepage(request):
     return JSONResponse({'hello': 'world'})
 
+async def julian_endpoint(request):
+    date = str(datetime.now())
```
Comment (t1waz) [inline backend/main.py]: variable date is used only in one place so it can be directly used in dict

**Review by t1waz — CHANGES_REQUESTED** [review]
pls refactor

**Comment by t1waz** [comment]
pls tell rest of students that there is deadline with PR until next meet :)

**Comment by t1waz** [comment]
pls send this discord to other students (we will try to use it for communication) https://discord.gg/fpuqSQBk

**Comment by JulianKorgol** [comment]
I posted and sent the link. :D

**Review by t1waz — APPROVED** [review]
LGMT

**Pull request merged**